packages/block-editor/src/components/block-tools/block-selection-button.js

```diff
@@ -76,7 +76,7 @@ function BlockSelectionButton( { clientId, rootClientId, blockElement } ) {
 	const { setNavigationMode, removeBlock } = useDispatch( blockEditorStore );
 	const ref = useRef();
 
-	const blockType = getBlockType( name );
+	const blockType = getBlockType( name ) || 'core/missing';
```

Comment:
Let's not use 'core/missing' directly here, since those internal details may change. One option here would be to drop this change, but update `getAccessibleBlockLabel` to not error when blockType is undefined.

Reply:
Again, not sure. I remember ```getBlockType``` returns an object with properties like ``` title, attributes, icon``` passing string `core/missing` is not correct. Let me fix this as well.

Reply:
@gwwar, I have changed ```getAccessibleBlockLabel``` function definition. What do you think about this change? Looking forward to your inputs.
``` javascript
export function getAccessibleBlockLabel(
	blockType,
	attributes,
	position,
	direction = 'vertical'
) {
	// `title` is already localized, `label` is a user-supplied value.
	const title = blockType?.title;
	const label = blockType ? getBlockLabel( blockType, attributes, 'accessibility' ) : '';
	const hasPosition = position !== undefined;

	// getBlockLabel returns the block title as a fallback when there's no label,
	// if it did return the title, this function needs to avoid adding the
	// title twice within the accessible label. Use this `hasLabel` boolean to
	// handle that.
	const hasLabel = label && label !== title;

	if ( hasPosition && direction === 'vertical' ) {
		if ( hasLabel ) {
			return sprintf(
				/* translators: accessibility text. 1: The block title. 2: The block row number. 3: The block label.. */
				__( '%1$s Block. Row %2$d. %3$s' ),
				title,
				position,
				label
			);
		}

		return sprintf(
			/* translators: accessibility text. 1: The block title. 2: The block row number. */
			__( '%1$s Block. Row %2$d' ),
			title,
			position
		);
	} else if ( hasPosition && direction === 'horizontal' ) {
		if ( hasLabel ) {
			return sprintf(
				/* translators: accessibility text. 1: The block title. 2: The block column number. 3: The block label.. */
				__( '%1$s Block. Column %2$d. %3$s' ),
				title,
				position,
				label
			);
		}

		return sprintf(
			/* translators: accessibility text. 1: The block title. 2: The block column number. */
			__( '%1$s Block. Column %2$d' ),
			title,
			position
		);
	}

	if ( hasLabel ) {
		return sprintf(
			/* translators: accessibility text. %1: The block title. %2: The block label. */
			__( '%1$s Block. %2$s' ),
			title,
			label
		);
	}

	return sprintf(
		/* translators: accessibility text. %s: The block title. */
		__( '%s Block' ),
		title
	);
}
```